package-lock.json

```diff
@@ -16519,6 +16520,11 @@
         "node": ">=6"
       }
     },
+    "node_modules/pako": {
+      "version": "1.0.11",
+      "resolved": "https://registry.npmjs.org/pako/-/pako-1.0.11.tgz",
+      "integrity": "sha512-4hLB8Py4zZce5s4yd9XzopqwVv/yGNhV1Bl8NTmCq1763HeK2+EwVTv+leGeL13Dnh2wfbqowVPXCIO0z4taYw=="
+    },
```

Comment:
is this really needed?

Reply:
it's used by UTIF 
https://github.com/photopea/UTIF.js/blob/master/package.json#L22
https://github.com/photopea/UTIF.js/blob/master/UTIF.js#L44

Reply:
gotcha